resources/js/hiroapp/components/recruiter/CreatePosition.jsx

```diff
@@ -0,0 +1,123 @@
+import axios from "axios";
+import React, { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+const CreatePosition = () => {
+	const navigate = useNavigate;
+
+	const [positions, setPositions] = useState(null);
+	const [values, setValues] = useState({
+		name: "",
+		description: "",
+		start_date: "",
+		end_date: "",
+		pay_grade: "",
+	});
+
+	const fetchPositions = async () => {
+		try {
+			const response = await axios.get("/api/positions/all");
+			setPositions(response.data);
+		} catch (error) {
+			console.log(error.response.data);
+		}
+	};
+
+	const handleChange = (ev) => {
+		setValues((previous_values) => {
+			return {
+				...previous_values,
+				[ev.target.name]: ev.target.value,
+			};
+		});
+	};
+
+	const handleSubmit = async (ev) => {
+		ev.preventDefault();
+		try {
+			const response = await axios.post("/api/positions", values);
+			navigate("/");
```

Comment:
you can redirect user to the created position so it's more user friendly and you don't need to clean the form afterwards:
so you can use something like:
navigate("/positions/" + response.data.id);

Reply:
Of course you are right